### ISSUE TITLE
Unable to setup `CoroutineLaunchedInTestWithoutRunTest` rule

### ISSUE THREAD

**Issue by piloudu** [detekt/detekt#7338]
## Expected Behavior
The following YAML configuration should enable the `CoroutineLaunchedInTestWithoutRunTest` rule:

```YAML
coroutines:
  active: true
  CoroutineLaunchedInTestWithoutRunTest:
    active: false
  # ...
```

## Observed Behavior
Enabling the `CoroutineLaunchedInTestWithoutRunTest` rule with the same YAML config as in here
https://github.com/detekt/detekt/blob/f8bf79b24baebd10fda01ffe2ef586cf8bc1fff1/detekt-core/src/main/resources/default-detekt-config.yml#L177

produces the following error:

```
Property 'coroutines>CoroutineLaunchedInTestWithoutRunTest' is misspelled or does not exist
```

The rule is pretty new

## Steps to Reproduce
Happens both on `v1.23.5` and `v1.23.6`

## Your Environment
Detekt dependencies in use in `libs.versions.toml`:

```TOML
[versions]
detekt="1.23.6"

[libraries]
detekt-api = { module = "io.gitlab.arturbosch.detekt:detekt-api", version.ref = "detekt" }
detekt-formatting = { module = "io.gitlab.arturbosch.detekt:detekt-formatting", version.ref = "detekt" }
detekt-test = { module = "io.gitlab.arturbosch.detekt:detekt-test", version.ref = "detekt" }

[plugins]
detekt = { id = "io.gitlab.arturbosch.detekt", version.ref = "detekt" }
```

**Comment by piloudu** [comment]
> That's because the rule is available only on `main` and hasn't been released yet: [f8d0942](https://github.com/detekt/detekt/commit/f8d09429e1f24619c3a043f751da9dab9c30bf10)

Thanks for your answer @cortinico. I'm having a hard time at understanding why this rule is not already released, as PR #6692 was merged to `main` on Jan 26th and [the latter release branch](https://github.com/detekt/detekt/releases/tag/v1.23.6) was created on Mar 25th

Can you help me understand this?

**Comment by cortinico** [comment]
> and [the latter release branch](https://github.com/detekt/detekt/releases/tag/v1.23.6) was created on Mar 25th

Because the 1.23.x release are happening from the `release/1.x` branch: https://github.com/detekt/detekt/tree/release/1.x
